### ISSUE TITLE
(Question) Fine Tuning to Transfer Learning Transition

### ISSUE THREAD

**Issue by silvernine209** [AlexeyAB/darknet#1444]
I was wondering if I can use my fine tuning weights `yolo-obj_fine_tuning_50300.weights` (still ongoing and training) and switch it to transfer learning by removing `stopbackward=1` from my `.cfg` file. 
I understand it's the best to start scratch again for transfer learning, but I already put in days to train my weights up to 50300.

Below is **iteration vs mAP**  for my 1.6 million train images and 500 classes fine tuning learning.

![image](https://user-images.githubusercontent.com/37491067/44305391-36f61280-a344-11e8-8e6e-cf7a4157ba3a.png)

**Comment by AlexeyAB** [comment]
Yes, you can try to remove `stopbackward=1` and continue training.

----

You must train from the begining only if you changed number of: classes, anchors, filters, layers, or type of layers.

**Comment by silvernine209** [comment]
Thank you for the clarification!
